mobius-android/src/main/java/com/spotify/mobius/android/MobiusLoopViewModel.java

```diff
@@ -70,14 +70,15 @@
   private final AtomicBoolean loopActive = new AtomicBoolean(true);
 
   protected MobiusLoopViewModel(
-      @Nonnull Function<Consumer<V>, MobiusLoop.Factory<M, E, F>> loopFactoryProvider,
+      @Nonnull MobiusLoopFactoryProvider<M, E, F, V> loopFactoryProvider,
```

Comment:
This (changing the constructor signature) breaks binary compatibility, so we can't do it this way. Unfortunately, our japicmp checks aren't applied to mobius-android, or the build would have failed (I forget why we're not doing that, probably we were not able to get it to work).

Reply:
So I'll just introduce a new constructor then?